src/components/images/ImagePreviewOverlay.tsx

```diff
@@ -144,6 +144,8 @@ function ImagePreview({
         disableAnimations={false}
         height={height}
         horizontalPadding={0}
+        onZoomIn={undefined}
+        onZoomOut={undefined}
```

Comment:
@jxom this was the only big change into our brach, there was a fix https://github.com/rainbow-me/rainbow/pull/3150 that added two additional methods to this component that we're missing now

Reply:
seems fine

Reply:
ok merging!